pkg/channel/distributed/controller/env/environment_test.go

```diff
@@ -94,16 +105,16 @@ func TestGetEnvironment(t *testing.T) {
 	testCase.expectedError = getMissingRequiredEnvironmentVariableError(ReceiverImageEnvVarKey)
 	testCases = append(testCases, testCase)
 
+	testCase = getValidTestCase("Invalid Config - ResyncPeriodMinutes")
+	testCase.resyncPeriodMinutes = "NAN"
+	testCase.expectedError = getInvalidIntEnvironmentVariableError(testCase.resyncPeriodMinutes, env.ResyncPeriodMinutesEnvVarKey)
+	testCases = append(testCases, testCase)
+
```

Comment:
Missing an Invalid Int test case ?  (I think there are others to that effect in that test already?)

Reply:
I added some more test cases for checking the default value for the optional variable; NAN is already an invalid int test, (and valid int is already checked in every other test case)